src/components/ArticleCard.jsx

```diff
@@ -13,6 +15,34 @@ const ArticleCard = ({
     votes,
   },
 }) => {
+  const [voteCount, setVoteCount] = useState(votes);
+  async function increaseVote() {
+    try {
+      setVoteCount((prevVote) => {
+        return prevVote + 1;
+      });
+      await increaseArticleVote(article_id);
+    } catch (err) {
+      console.log(err);
+      setVoteCount(() => {
+        return votes;
+      });
+      window.alert("There was an error");
+    }
+  }
+  async function decreaseVote() {
+    try {
+      setVoteCount((prevVote) => {
+        return prevVote - 1;
+      });
+      await decreaseArticleVote(article_id);
+    } catch (err) {
+      console.log(err);
```

Comment:
Console.log

Reply:
I was told console.log for errors are allowed